### DISCUSSION TITLE
current state node - define entities with all possible options

### DISCUSSION THREAD

**Discussion by Hadatko** [zachowj/node-red-contrib-home-assistant-websocket#1257] [Feature Request]
Hello it looks like i can define just one entity in current state node. I would like to sent request to several entities and then each entity will be continue in its thread further.

Basically it is like there is one trigger, then i need add several current state nodes where i fill entities, and then these entities continue further. It would be nice to merge them with regex into one node.

FROM:
```
          ------- E1----
       /                  \
T----- ---------E2--------- ------ F
       \                  /
          -------E3------
```
           
TO:
```
T-----------E(trigger E1, E2, E3) ------F (each En will continue by path on its own)
```

**Comment by zachowj** [comment]
Take a look at the get-entities node.

**Comment by Hadatko** [comment]
So if i use this node and split option and in next node {{entity_id}} it will have same behavior? Thank you very much
![image](https://github.com/zachowj/node-red-contrib-home-assistant-websocket/assets/13130700/fa7ca223-73c5-4c44-927c-af9b9d914ebc)

**Reply by zachowj** [reply]
The get-entities node when split is selected as the output method. It will output every entity that matches the conditions as a new message. There will be no need for the current state node.

**Comment by Hadatko** [comment]
I think we understand each other. The reason why i kept is because i am using condition inside that. Now the node can be replaced with switch.
